src/Service.Tests/Unittests/ConfigFileWatcherUnitTests.cs

```diff
@@ -159,5 +198,164 @@ public void HotReloadConfigRestRuntimeOptions()
                 File.Delete(configName);
             }
         }
+
+        /// <summary>
+        /// Creates a hot reload scenario in which the schema is wrong which causes
+        /// hot reload to fail, then we check that the program is still able to work
+        /// properly by showing us that it is still using the same configuration file
+        /// from before the hot reload.
```

Comment:
- "hot reload scenario in which the schema is wrong" clarify what you mean by "wrong"

/// properly by validating that the DAB engine is still using the same configuration file
/// from before the hot reload.